src/Cli/dotnet/commands/dotnet-workload/install/NetSdkManagedInstaller.cs

```diff
@@ -138,7 +140,77 @@ public void RollBackWorkloadPackInstall(PackInfo packInfo, SdkFeatureBand sdkFea
             }
         }
 
-        public void InstallWorkloadManifest(ManifestId manifestId, ManifestVersion manifestVersion, SdkFeatureBand sdkFeatureBand) => throw new NotImplementedException();
+        public void InstallWorkloadManifest(ManifestId manifestId, ManifestVersion manifestVersion, SdkFeatureBand sdkFeatureBand)
+        {
+            string packagePath = null;
+            string tempExtractionDir = null;
+            string tempBackupDir = null;
+            var manifestPath = Path.Combine(_dotnetDir, "sdk-manifests", sdkFeatureBand.ToString(), manifestId.ToString());
+
+            _reporter.WriteLine(string.Format(LocalizableStrings.InstallingWorkloadManifest, manifestId, manifestVersion));
+
+            try
+            {
+                TransactionalAction.Run(
+                   action: () =>
+                   {
+                       packagePath = _nugetPackageDownloader.DownloadPackageAsync(new PackageId(manifestId.ToString()), new NuGetVersion(manifestVersion.ToString())).Result;
```

Comment:
@dsplaisted @wli3 @joeloff in case you have feedback. 

I haven't been able to test this end to end because some of the specs are conflicting about how to translate between manifest ids/ versions and package ids/versions. For now I'm assuming the package id and version will match the manifest id and version, but let me know if that is not right. Also, none of the existing manifest packages are following any of these patterns so we'll need to sync with them.

Reply:
The latest plan of the record is package version is the same as the manifest version and it is a string ("6.0.1xx")

@mhutch @dsplaisted please update the specs to reflect this decision